Popular Movies/Popular Movies/Controllers/MovieDetailTableViewController.m

```diff
@@ -150,14 +153,30 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
     if([collectionView isKindOfClass:[ActorCollectionView class]]){
         ActorCollectionViewCell* cellOneActor = (ActorCollectionViewCell*)[collectionView dequeueReusableCellWithReuseIdentifier:actorReuseIdentifier forIndexPath:indexPath];
         Actor* singleActor = [[Actor alloc]init];
-        if(castCollection.cast) singleActor = (Actor*)[castCollection.cast objectAtIndex:indexPath.row];
+        if(selectedMovie.credits.cast) singleActor = (Actor*)[selectedMovie.credits.cast objectAtIndex:indexPath.row];
         [cellOneActor setUpActorCellWithActor:singleActor];
         return cellOneActor;
     }
     return cell;
 }
 -(CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath{
-    if(indexPath.section == 1){
+    if([tableView isKindOfClass:[ReviewsTableView class]]){
+        return 200.0f;
+    }
+    if(indexPath.section == 0) return 420.0f;
+    else if(indexPath.section == 1){
```

Comment:
This isn't really clean approach. We should use **UITableViewAutomaticDimension** to calculate cell height.